addons/website_sale/views/product_product_add.xml

```diff
@@ -12,6 +12,9 @@
             <xpath expr="//field[@name='company_id']" position="before">
                 <field name="website_url" invisible="1"/>
             </xpath>
+            <xpath expr="//div[@name='tax_info']" position="after">
+                <field name="public_categ_ids" string="Website Category" widget="many2many_tags" placeholder="Unpublished"/>
```

Comment:
can't we add `oe_inline` here to make width of this field same as other field?
same for pos_category :thinking:

Reply:
Yes we should apply I have checked by applying it but the field's width was same as of other fields even before so do we need to add that if it automatically adapt it ?